style.css

```diff
@@ -0,0 +1,206 @@
+:root {
+  --red: #f72210;
+  --yellow: #ffe400;
+  --body-background: #f4f5f6;
+}
```

Comment:
Kudos for using variables :clap:

Reply:
Thank you Sina Sevgi, 
We are trying to do our best